lib/cisco_node_utils/command_reference.rb

```diff
@@ -416,6 +416,11 @@ def supports?(feature)
       !(value.is_a?(UnsupportedCmdRef) || value.nil?)
     end
 
+    def property_supported?(feature, property)
```

Comment:
Consider extending the `supports?` method instead of creating a new separate one:

```ruby
def supports?(feature, property=nil)
  value = @hash[feature]
  value = value[property] if value && property
  !(value.is_a(UnsupportedCmdRef) || value.nil?)
end
```